Source/Entities/AEmitter.cpp

```diff
@@ -339,6 +339,7 @@ void AEmitter::SetFlash(Attachable* newFlash) {
 		m_pFlash->SetInheritsRotAngle(false);
 		m_pFlash->SetDeleteWhenRemovedFromParent(true);
 		m_pFlash->SetCollidesWithTerrainWhileAttached(false);
+		m_pFlash->MovableObject::SetEffectDrawEveryFrame(false);
```

Comment:
please remove the manual disambiguation of MovableObject:: - this shouldn't be necessary.